pkgs/by-name/me/mealie/package.nix

```
@@ -0,0 +1,100 @@
+{ lib, callPackage, fetchFromGitHub, gnused, nixosTests, python3, python3Packages, writeShellScript }: let
+  version = "1.0.0-RC2";
+  src = fetchFromGitHub {
+    owner = "mealie-recipes";
+    repo = "mealie";
+    rev = "v${version}";
+    sha256 = "sha256-/sht8s0Nap6TdYxAqamKj/HGGV21/8eYCuYzpWXRJCE=";
+  };
+
+  frontend = callPackage (import ./mealie-frontend.nix src version) { };
+
+  # TODO  Add crfpp in runtime dependencies for ingredient quantities NLP
+
+in python3Packages.buildPythonPackage rec {
+  pname = "mealie";
+  inherit version src;
+  format = "pyproject";
+
+  patches = [
+    ./mealie_statedir.patch
+    ./mealie_init_db.patch
+    ./mealie_logger.patch
+  ];
+
+  nativeBuildInputs = [
+    python3Packages.poetry-core
+    python3.pkgs.pythonRelaxDepsHook
+  ];
+
+  pythonRelaxDeps = true;
+
+  propagatedBuildInputs = with python3Packages; [
+    aiofiles
+    alembic
+    aniso8601
+    appdirs
+    apprise
+    bcrypt
+    extruct
+    fastapi
+    gunicorn
+    jinja2
+    lxml
+    orjson
+    passlib
+    pillow
+    psycopg2
+    pyhumps
+    pytesseract
+    python-dotenv
+    python-jose
+    python-ldap
+    python-multipart
+    python-slugify
+    pyyaml
+    rapidfuzz
+    recipe-scrapers
+    sqlalchemy
+    uvicorn
+  ];
+
+  doCheck = true;
+
+  postInstall = let
+    start_script = writeShellScript "start-mealie" ''
+      export STATIC_FILES="${frontend}"
+      ${python3Packages.gunicorn}/bin/gunicorn "$@" -k uvicorn.workers.UvicornWorker mealie.app:app;
+    '';
+  in ''
+    mkdir -p $out/config $out/bin
+    ${lib.getExe gnused} 's+script_location = alembic+script_location = ${src}/alembic+g' ${src}/alembic.ini > $out/config/alembic.ini
```

Use [`substitute`](https://nixos.org/manual/nixpkgs/unstable/#fun-substitute)/[`substituteInPlace`](https://nixos.org/manual/nixpkgs/unstable/#fun-substituteInPlace) instead of `sed`:

```suggestion
    substitute ${src}/alembic.ini $out/config/alembic.ini \
      --replace 'script_location = alembic' 'script_location = ${src}/alembic'
```